### ISSUE TITLE
How can we fine-tune Pi3 for different domain images?

### ISSUE THREAD

**Issue by ShntrIto** [yyfz/Pi3#147]
Hi authors,
Thank you for sharing this great work!

I am interested in applying your work to out-of-domain images (e.g., medical images) and would like to fine-tune the model rather than training from scratch. However, looking at the training branch, the provided code appears to be for training from scratch, and I could not find any fine-tuning-specific scripts or configurations.
Could you provide any guidance on how to fine-tune the model on a different image domain?
Thank you!

**Comment by yyfz** [comment]
Hi, [ShntrIto](https://github.com/ShntrIto)! You can simpliy initial the model with Pi3 weight and use lower learning rate.

**Comment by ShntrIto** [comment]
Hi @yyfz ! Thank you so much for the guidance! I will try it out and let you know if I have any further questions.
